src/blocks/stars.css

```diff
@@ -0,0 +1,37 @@
+/* #region stars */
+.stars-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  display: inline-flex;
+  width: 16px;
+  height: 16px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars-container__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars__star--active {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
need to transfer all styles for stars to keep the ability to change the number of active stars by changing the modifier